src/main/webapp/app/shared/table/OncoKBTable.scss

```diff
@@ -1,9 +1,53 @@
 .oncokb-table {
   word-wrap: break-word;
+  border: 1px solid rgba(0, 0, 0, 0.1);
```

Comment:
why we do `rgba` here?

Reply:
I'm trying to get a different shade of gray. Mostly copied the styles from public website.